packages/kbn-esql-ast/src/ast_helpers.ts

```diff
@@ -120,11 +134,20 @@ export function textExistsAndIsValid(text: string | undefined): text is string {
 
 export function createLiteral(
   type: ESQLLiteral['literalType'],
-  node: TerminalNode | undefined
-): ESQLLiteral | undefined {
+  node: TerminalNode | null
+): ESQLLiteral {
   if (!node) {
-    return;
+    return {
```

Comment:
Why this change?

Reply:
Similar story to https://github.com/elastic/kibana/pull/185904#discussion_r1643188227